src/meta/test/server_state_test.cpp

```diff
@@ -1,8 +1,15 @@
 #include <gtest/gtest.h>
+
```

Comment:
As the comments shows, I recommend you adding your unit test in `src/meta/test/meta_app_operation_test.cpp`.